### PR TITLE
Pro Add-In ExecuteCommand NoArgs

### PR DESCRIPTION
**Pull request by llam001**
PR to address issue with Open Pro Project Items step.

### CODE REVIEW

File: ServiceBased/ProAddIn/WorkflowManagerSampleAddIn/Module1.cs [hunk 1]
```diff
@@ -41,7 +41,31 @@ protected override bool CanUnload()
...
+                {
+                    // Run the command specified by the id
+                    IPlugInWrapper wrapper = FrameworkApplication.GetPlugInWrapper(id);
+                    var command = wrapper as ICommand;
+                    if ((command != null) && command.CanExecute(null))
+                        command.Execute(null);
+                }
+                catch (System.Exception e)
+                {
+                    // ArcGIS.Desktop.Framework.Dialogs.MessageBox.Show($"ERROR: {e}", "Error running command");
```
Comment (AngelVDelgadillo) [inline ServiceBased/ProAddIn/WorkflowManagerSampleAddIn/Module1.cs]: Should this be commented out? If so, is it better just to remove it?

**Review by samuellabrador11235 — APPROVED** [review]
Changes look good to me, no comments

**Review by AngelVDelgadillo — APPROVED** [review]

File: ServiceBased/ProAddIn/WorkflowManagerSampleAddIn/Module1.cs [hunk 2]
```diff
@@ -41,7 +41,31 @@ protected override bool CanUnload()
...
+        /// <param name="id">The DAML control identifier.</param>
+        /// <returns>A user defined function that will execute asynchronously when invoked.</returns>
+        protected override Func<Task> ExecuteCommand(string id)
+        {
+            return () => QueuedTask.Run(() =>
+            {
+                try
+                {
+                    // Run the command specified by the id
+                    IPlugInWrapper wrapper = FrameworkApplication.GetPlugInWrapper(id);
```
Comment (kbedel) [inline ServiceBased/ProAddIn/WorkflowManagerSampleAddIn/Module1.cs:57]: Do we want to look up any command like this, or should be delegating to specific commands defined in this module?

Comment (llam001) [reply ServiceBased/ProAddIn/WorkflowManagerSampleAddIn/Module1.cs:57]: @kbedel - If the command specified by the id is not defined in this module, then this method will not be called.

Comment (kbedel) [reply ServiceBased/ProAddIn/WorkflowManagerSampleAddIn/Module1.cs:57]: Is this how we'd want users to actually implement this though? Or would we expect more like what Editing does here: https://devtopia.esri.com/ArcGISPro/ArcGISPro/blob/main/ArcGISDesktop/ArcGISEditing/ArcGIS.Desktop.Editing/EditingModule.cs#L5805-L5806

Comment (llam001) [reply ServiceBased/ProAddIn/WorkflowManagerSampleAddIn/Module1.cs:57]: Chatted with @kbedel and this is fine for the current implementation. We'll update samples again when we make changes in Pro 3.4.

**Review by kbedel — APPROVED** [review]

**Pull request merged**